src/main/frontend/multi-pipeline-graph-view/multi-pipeline-graph/main/SingleRun.tsx

```diff
@@ -24,22 +24,26 @@ export const SingleRun: (data: Props) => JSX.Element = ({ run }) => {
   }
 
   const handleNodeClick = (nodeName: string, id: number) => {
-    console.log(nodeName, id);
     let redirect = `../${run.id}/pipeline-console?selected-node=${id}`;
     if (onJobView) {
       redirect = `${run.id}/pipeline-console?selected-node=${id}`;
     }
     window.location.href = redirect;
   };
+
+  const jobStatus = `${run.result.toLowerCase()}`;
+
   return (
     <tr>
       <td>
-        <a
-          href={singleRunPage}
-          className="jenkins-table__link pgw-user-specified-text"
-        >
-          {run.displayName}
-        </a>
+        <div className="PWGx-PipelineGraph-Summary-container">
+          <a href={singleRunPage} className={`PWGx-Link  ${jobStatus}`}>
+            <p>{run.displayName}</p>
+          </a>
+          <div className="PWGx-Start">
+            <span>{run.startTime} - {jobStatus === "in_progress" ? "Running..." : run.duration}</span>
```

Comment:
I think it would be good to move this out to its own 'chip' but I wasn't able to make it work in the 10mins or so I had (the display flex column wants each on their own line)

this is probably ok though

<img width="1011" alt="image" src="https://github.com/jenkinsci/pipeline-graph-view-plugin/assets/21194782/87110de6-ffe6-4463-b72e-bba5a4db571b">